### PR TITLE
Various fixes related to the webprod/webprod2 to blue/green change

### PR DESCRIPTION
**Pull request by netsettler**
A lot of these were judgment calls, so please feel free to suggest whether you agree with the choices I made.

Also, I started this PR before testing was done, so it's possible the tests won't pass. But I figured it was close enough even so that it was worth getting feedback going.

### CODE REVIEW

**Comment by j1z0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/encoded/types/user.py  1
         

Complexity decreasing per file
==============================
+ src/encoded/commands/load_data.py  -1
+ src/encoded/tests/test_clear_db_es_contents.py  -4
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5303766&bid=17360012)

File: deploy/tests/test_generate_production_ini.py [hunk 1]
```diff
@@ -14,6 +14,13 @@
     template_environment_names,
 )
 
+
+# TODO: Maybe this should move to env_utils? If not, at least to a non-test file.
+#       Then again, if we used the "single parameterized ini file" we could side-step that. -kmp 3-Apr-2020
+
+FOURFRONT_DEPLOY_NAMES = ['hotseat', 'mastertest', 'webdev', 'webprod', 'webprod2']
```
Comment (willronchetti) [inline deploy/tests/test_generate_production_ini.py:21]: Shouldn't this include blue and green?

Comment (netsettler) [reply deploy/tests/test_generate_production_ini.py:21]: Yes. Although I don't think these tests are even running by default. I'm changing `pytest.ini`, too. Please double-check that.

**Comment by j1z0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/encoded/types/user.py  1
         

Complexity decreasing per file
==============================
+ src/encoded/tests/test_clear_db_es_contents.py  -4
+ src/encoded/commands/load_data.py  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5303766&bid=17360012)

**Comment by netsettler** [comment]
I'll go ahead and merge this.

**Pull request merged**